src/backend/cuda/hooks/yaksuri_cuda_init_hooks.c

```diff
@@ -141,8 +141,7 @@ int yaksuri_cuda_init_hook(yaksur_gpudriver_hooks_s ** hooks)
         }
     }
 
-    yaksuri_cudai_global.stream = (cudaStream_t *)
-        calloc(yaksuri_cudai_global.ndevices, sizeof(cudaStream_t));
+    yaksuri_cudai_global.streams = calloc(yaksuri_cudai_global.ndevices, sizeof(cudai_stream));
```

Comment:
we can convert this to regular `malloc` now since we track the state elsewhere

Reply:
`cudai_stream` is an internal struct (pair) that includes the boolean flag. If we use malloc, then we need init the flags to 0 separately.

Reply:
Oh, right. Forgot the boolean was embedded along with the stream handle. Carry on.